### ISSUE TITLE
UX meeting agenda 14 June 2016

### ISSUE THREAD

**Issue by stevencl** [Microsoft/vscode#7663]
- Tabs/Stacks: decide on settings #7643
- Find widget does not blend well when tabs enabled #7649
- Wrong input box color in dark theme #7648
- Discussion on tabs
- Position of debug toolbar #4580 
- Search and replace UX mockups

**Comment by isidorn** [comment]
- Position of debug toolbar https://github.com/Microsoft/vscode/issues/4580

**Comment by bpasero** [comment]
* Find and Replace exploration https://github.com/Microsoft/vscode/issues/7667

**Comment by stevencl** [comment]
I've added notes to each of the items referenced above.

In addition, we discussed the scroll behaviour of the tabs when in overflow. In particular, we looked at how we can indicate that there are tabs to the left and right that the user can scroll. Ben demonstrated the use of a shadow (consistent with how we use shadows elsewhere). @bgashler1 - we should talk about this and how we can finesse the look.

We also talked about whether or not we should reduce the height of the tabs. They are the same height as the viewlet title bar which I think ends up making the tabs look a little heavy. We didn't reach any conclusion on this though.

Lastly we experimented with adding the border colour to the left hand side of the selected tab but with the problems we have with the current colours of the selected tab, unselected tabs and the tab well, we didn't think that the left border helped much.
